sbt-ash/src/main/scala/org/virtuslab/ash/AkkaSerializationHelperKeys.scala

```diff
@@ -0,0 +1,21 @@
+package org.virtuslab.ash
+
+import sbt.File
+import sbt.librarymanagement.ModuleID
+import sbt.settingKey
+import sbt.taskKey
+
+trait AkkaSerializationHelperKeys {
+  lazy val ashDumpEventSchemaCompilerPlugin = settingKey[ModuleID]("")
+  lazy val ashRegistrationCheckerCompilerPlugin = settingKey[ModuleID]("")
+  lazy val ashSerializabilityCheckerCompilerPlugin = settingKey[ModuleID]("")
+
+  lazy val ashCompilerPluginVersion = settingKey[String]("Sets the version of compiler plugin")
```

Comment:
Hmmm shouldn't it be plural `ashCompilerPlugins...`? 🤔 since singular `Plugin` kinda creates an impression that there is well, just one of them ;)

Reply:
This is more complicated.
 Accodring to the sbt keys convention, I define keys widely and consume them narrowly. So setting `ashCompilerPluginVersion = ""` sets for all plugins while `ashSerializabilityCheckerCompilerPlugin / ashCompilerPluginVersion = ""` sets only for  one plugin.

Reply:
Oh okay, I see 🤔